AdyenDropIn/DropInComponent.swift

```diff
@@ -14,7 +14,10 @@ import Adyen
 import UIKit
 
 /// A component that handles the entire flow of payment selection and payment details entry.
-public final class DropInComponent: NSObject, PresentableComponent {
+public final class DropInComponent: NSObject,
+    PresentableComponent,
+    PaymentAwareComponent,
+    LoadingComponent {
```

Comment:
We can probably remove this. Loading state of DropIn controlled by DropIn itself